tests/fixtures/entity-viewer/transcript.ts

```diff
@@ -22,15 +22,19 @@ import { createProduct } from './product';
 import { createExternalReference } from './external-reference';
 import { getFeatureCoordinates } from 'src/content/app/entity-viewer/shared/helpers/entity-helpers';
 
-import { FullTranscript } from 'src/shared/types/thoas/transcript';
-import { Exon, SplicedExon, PhasedExon } from 'src/shared/types/thoas/exon';
-import { Slice } from 'src/shared/types/thoas/slice';
-import { FullCDS } from 'src/shared/types/thoas/cds';
-import { CDNA } from 'src/shared/types/thoas/cdna';
-import { FullProductGeneratingContext } from 'src/shared/types/thoas/productGeneratingContext';
-import { ProductType } from 'src/shared/types/thoas/product';
-import { ExternalReference } from 'src/shared/types/thoas/externalReference';
-import { TranscriptMetadata } from 'src/shared/types/thoas/metadata';
+import type { FullTranscript } from 'src/shared/types/thoas/transcript';
+import type {
+  Exon,
+  SplicedExon,
+  PhasedExon
+} from 'src/shared/types/thoas/exon';
+import type { Slice } from 'src/shared/types/thoas/slice';
+import type { FullCDS } from 'src/shared/types/thoas/cds';
+import type { CDNA } from 'src/shared/types/thoas/cdna';
+import type { FullProductGeneratingContext } from 'src/shared/types/thoas/productGeneratingContext';
+import { ProductType, type Product } from 'src/shared/types/thoas/product';
+import type { ExternalReference } from 'src/shared/types/thoas/externalReference';
+import type { TranscriptMetadata } from 'src/shared/types/thoas/metadata';
```

Comment:
Do we need to consider putting all these `thoas` types into a `index.ts` file?

Reply:
🤷‍♂️  I don't have a very strong feeling about this, but I don't have a particular desire to do that either :-) Not in this PR anyway.